base/bucket_n1ql_test.go

```diff
@@ -575,3 +574,37 @@ func tearDownTestIndex(bucket *CouchbaseBucketGoCB, indexName string) (err error
 	}
 
 }
+
+func TestWaitForBucketExistence(t *testing.T) {
+	if UnitTestUrlIsWalrus() {
+		t.Skip("This test only works against Couchbase Server")
+	}
+
+	testBucket := GetTestBucket(t)
+	defer testBucket.Close()
+	bucket, ok := testBucket.Bucket.(*CouchbaseBucketGoCB)
+	assert.True(t, ok, "Requires gocb bucket")
+
+	// Create index
+	indexName, expression, filterExpression := "index1", "_sync", ""
```

Comment:
It's quite a bit harder to read these 3 variable declarations when all in one line, and they're being used like constants anyway.

```suggestion
const (
	indexName        = "index1"
	expression       = "_sync"
	filterExpression = ""
)
```

Reply:
This change has been committed to the latest revision of code.